services/backend/src/core/profile/profile.js

```diff
@@ -204,8 +208,8 @@ async function updateProfile(request, response) {
       await prisma.user.update({
         where: { id: userId },
         data: {
-          firstName,
-          lastName,
+          firstName: firstName ? capitalizeFirstLetter(firstName) : undefined,
```

Comment:
If we set it to undefined wouldnt that erase the name from the db ?

Reply:
No, it would just act as there would be no variable there. If its null, it will make it null in the db